### PR TITLE
[Fixes AB#659] UI fixes

### PR DESCRIPTION
**Pull request by Skafec**
Images now have cover object-fit property. This property would sometimes make images sharp so I added blur filter
to reduce the overall sharpness and make it more pleasant to watch (investment groups screen).

Other bugs were fixed according to the task description.

### CODE REVIEW

File: src/app/wallet/wallet.component.html [hunk 1]
```diff
@@ -168,7 +155,7 @@ <h4 class="init-left__title">
                                                 <span class="info__state-failed" *ngIf="transaction.state === 'FAILED'">
                                                     Failed
                                                 </span>
-                                                {{ transaction.date | date:'medium' }}
+                                                {{ transaction.date | date: "medium" }}
```
Comment (mightymatth) [inline src/app/wallet/wallet.component.html:158]: Revert this back. Use convention as Angular community uses it (look [here](https://angular.io/guide/pipes#transforming-data-with-parameters-and-chained-pipes)).

File: src/styles/_common.scss [hunk 2]
```diff
@@ -127,6 +127,10 @@
   display: contents;
 }
 
+.uppy-Dashboard-AddFiles-info {
+  display: inline-block;
+}
+
```
Comment (mightymatth) [inline src/styles/_common.scss:133]: This does not belong to common stuff. If this is the first case of overriding something that already exists, create _overrides.scss partial and throw it there.

File: src/app/organizations/manage-organizations/manage-organizations.component.scss [hunk 3]
```diff
@@ -58,7 +58,8 @@
       border-radius: 1px;
     }
     &__img img {
-      object-fit: fill;
+      object-fit: cover;
+      filter: blur(0.5px);
```
Comment (mightymatth) [inline src/app/organizations/manage-organizations/manage-organizations.component.scss:62]: why this? just asking.

**Review by mightymatth — CHANGES_REQUESTED** [review]

File: src/app/wallet/wallet.component.html [hunk 4]
```diff
@@ -145,7 +145,7 @@ <h4 class="init-left__title">Your private, secure wallet</h4>
                                                       *ngIf="transaction.state === transactionState.MINED">
                                                     <span class="state__invested"
                                                           *ngIf="transaction.type === transactionType.INVEST">
-                                                        {{ transaction.share | percent: "1.0-10" }} Invested —
+                                                        {{ transaction.share | percent: '1.0-10' }} Invested —
```
Comment (mightymatth) [inline src/app/wallet/wallet.component.html]:
```suggestion
                                                        {{ transaction.share | percent:'1.0-10' }} Invested —
```

File: src/app/wallet/wallet.component.html [hunk 5]
```diff
@@ -155,7 +155,7 @@ <h4 class="init-left__title">Your private, secure wallet</h4>
                                                 <span class="info__state-failed" *ngIf="transaction.state === 'FAILED'">
                                                     Failed
                                                 </span>
-                                                {{ transaction.date | date: "medium" }}
+                                                {{ transaction.date | date: 'medium' }}
```
Comment (mightymatth) [inline src/app/wallet/wallet.component.html]:
```suggestion
                                                {{ transaction.date | date:'medium' }}
```

**Review by mightymatth — CHANGES_REQUESTED** [review]

File: src/app/organizations/manage-organizations/manage-organizations.component.scss (hunk 3 above)
Comment (mightymatth) [reply src/app/organizations/manage-organizations/manage-organizations.component.scss:62]: I understand the intent, but could you please try to upload a lower quality photo and see what will happen when putting a blur on it.

**Review by mightymatth — CHANGES_REQUESTED** [review]

File: src/app/wallet/wallet.component.html (hunk 4 above)
Comment (Skafec) [reply src/app/wallet/wallet.component.html]: This was fixed in the last commit (https://github.com/AMPnet/ampnet-crowdfunding-frontend/pull/152/commits/df33199816a7d0a612d26a23335a8005518b6a2c)

File: src/app/wallet/wallet.component.html (hunk 5 above)
Comment (Skafec) [reply src/app/wallet/wallet.component.html]: This was fixed in the last commit (https://github.com/AMPnet/ampnet-crowdfunding-frontend/pull/152/commits/df33199816a7d0a612d26a23335a8005518b6a2c)

File: src/app/organizations/manage-organizations/manage-organizations.component.scss (hunk 3 above)
Comment (Skafec) [reply src/app/organizations/manage-organizations/manage-organizations.component.scss:62]: Right, I understand what you mean. We can't be sure that users will provide high res images all the time. I'm going to remove this.

**Review by mightymatth — COMMENTED** [review]
Ready for QA

**Review by mightymatth — APPROVED** [review]

**Pull request merged**